pkgs/development/compilers/arocc/package.nix

```diff
@@ -21,7 +21,10 @@ stdenv.mkDerivation (finalAttrs: {
   passthru = {
     inherit zig;
     isArocc = true;
-    wrapped = wrapCCWith { cc = finalAttrs.finalPackage; };
+    wrapped = wrapCCWith {
+      cc = finalAttrs.finalPackage;
+      inherit (finalAttrs.passthru.zig.passthru.stdenv.cc) bintools;
+    };
```

Comment:
Explain in commit message?

Reply:
Do we need to explain the CC wrapper in the commit message?

Reply:
No, explain why this is being changed to use the zig bintools.

Reply:
Oh, I forgot why but I know it was needed for something lol.

Reply:
Turns out things are just fine without it, idk what wasn't working without it lol.